pages/national_resources/uk_resources.md

```diff
@@ -92,4 +92,4 @@ This page provides an overview of the data management resources in the UK. The t
   * The [Safepod Network](https://safepodnetwork.ac.uk/) providing approved researchers with access to sensitive data
   * [UK Biobank](https://www.ukbiobank.ac.uk/) large-scale biomedical database and research resource
   * [UK Data Archive](https://www.data-archive.ac.uk/) collection of social, economic and population data
-  * Clinical Practice Research Datalink [CPRD](https://cprd.com/) research service supporting retrospective and prospective public health and clinical studies
+  * Clinical Practice Research Datalink [CPRD](https://cprd.com/) supports public health and clinical studies in the UK, by maintaining a collection of anonymised health dataset representing UK population.
```

Comment:
I think this sentence is not correct: "a collection of anonymised health dataset representing UK population." @Nazeeefa 
Maybe: "a collection of anonymised health datasets representing the UK population."